en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-embedded-h2.md

```diff
@@ -42,16 +42,16 @@ The following sections describe how to replace the default H2 database with Embe
 -   [Creating database tables](#ChangingtoEmbeddedH2-Creatingdatabasetables)
 
 !!! warning
-H2 is not recommended in production
+    H2 is not recommended in production
 
-The embedded H2 database is NOT recommended in enterprise testing and production environments. It has lower performance, clustering limitations, and can cause file corruption failures. Please use an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, or MS SQL instead.
+    The embedded H2 database is NOT recommended in enterprise testing and production environments. It has lower performance, clustering limitations, and can cause file corruption failures. Please use an industry-standard RDBMS such as Oracle, PostgreSQL, MySQL, or MS SQL instead.
 
-You can use the embedded H2 database in development environments and as the local registry in a registry mount.
+    You can use the embedded H2 database in development environments and as the local registry in a registry mount.
 
 !!! tip
-Before you begin
+    Before you begin,
 
-You need to set up Embedded H2 before following the steps to configure your product with it. For more information, see [Setting up Embedded H2](https://docs.wso2.com/display/ADMIN44x/Setting+up+Embedded+H2) .
+    You need to set up Embedded H2 before following the steps to configure your product with it. For more information, see [Setting up Embedded H2](https://docs.wso2.com/display/ADMIN44x/Setting+up+Embedded+H2) .
```

Comment:
We should not be pointing to Admin docs https://docs.wso2.com/display/ADMIN44x/Setting+up+Embedded+H2
as that content is in the APIM docs already
For example:
https://apim.docs.wso2.com/en/latest/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-embedded-h2/

So this link should change as {{base_path}}/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-embedded-h2/